platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```
@@ -204,7 +202,9 @@ private File createFileFromAttachement(Attachment attachment, Response response)
                 try {
                     File uploadDir = new File(defaultFileLocation);
                     if (!uploadDir.exists()) {
-                        uploadDir.mkdirs();
+                        if(!uploadDir.mkdirs()) {
+                            LOGGER.warn("Unable to make directory");
```

Probably should throw an exception rather than log here since we try to copy into this directory later.